Makefile

```diff
@@ -284,13 +284,15 @@ test-update-goldenfiles: .init
 
 build-integration: .generate_files
 	$(DOCKER_CMD) go test -race github.com/kubernetes-sigs/service-catalog/test/integration/... -c
+	$(DOCKER_CMD) go test --tags="integration" -race github.com/kubernetes-sigs/service-catalog/pkg/controller/... -c
```

Comment:
we probably don't need to do this anymore. go caching should have improved everything enough that this doesn't save any time.

Need to ensure DOCKER_CMD mounts the cache dir, and we can drop the build ahead of running the tests part.

Reply:
I totally agree with you but maybe let's do that as the follow-up and remove the building from all the test stages (build-integration and build-e2e).